### PR TITLE
Refactor Qdrant Monitoring

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

This PR refactors Qdrant monitoring for improved parsing of Prometheus metrics and integration with Datadog. To support the new cluster setup, I renamed the environment variable to `QDRANT_NODES`, as we need to query each node independently.

## Deploy Plan

Set the environment variable `QDRANT_NODES`.

### CODE REVIEW

File: alerting/temporal/src/qdrant/index.ts [hunk 1]
```diff
@@ -7,22 +7,26 @@ import {
   GAUGE,
 } from "@datadog/datadog-api-client/dist/packages/datadog-api-client-v2/models/MetricIntakeType";
 
-const { QDRANT_CLUSTERS, QDRANT_MONITORING_API_KEY } = process.env;
+const { QDRANT_NODES, QDRANT_MONITORING_API_KEY } = process.env;
```
Comment (lasryaric) [inline alerting/temporal/src/qdrant/index.ts:10]: Can we use the new way of getting the configuration?

**Review by lasryaric — COMMENTED** [review]
Left one question.

**Review by spolu — APPROVED** [review]
Looks reasonable

File: alerting/temporal/src/qdrant/index.ts (hunk 1 above)
Comment (flvndvd) [reply alerting/temporal/src/qdrant/index.ts:10]: Fair ... we don't have the `types` module in this deployment and I did not want to add it for this, WDYT?

**Review by lasryaric — APPROVED** [review]
LGTM

**Pull request merged**